### DISCUSSION TITLE
Upload of folders to the external backend (S3 , GCS)

### DISCUSSION THREAD

**Discussion by michaelcpd** [drakkan/sftpgo#1192] [Ideas]
Hi guys, is there a possibility to recursively upload folders and not just files, to external backend (S3, GCS)?

**Comment by fntyler** [comment]
Could you provide more specifics? Is this through the client web-front end via HTTP using a SFTPGo Folder as a storage object?

**Comment by dogterbox** [comment]
Hi, I want this too, via client web (UI).
